### PR TITLE
Add verifyToken() to Backend SDK docs

### PR DESCRIPTION
**Pull request by alexisintech**
> [!IMPORTANT]
> 🔎 Previews:
>
> - https://clerk.com/docs/pr/1184/references/backend/verify-token

### Explanation:
The `verifyToken()` method was reported missing from the docs.

### This PR:

- adds the `verifyToken()` method to the Backend SDK docs

### CODE REVIEW

File: docs/references/backend/verify-token.mdx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+| `options` | `VerifyTokenOptions` | Options for verifying the token. |
+
+### `VerifyTokenOptions`
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `audience?` | `string \| string[]` | A string or list of [audiences](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3). |
+| `authorizedParties`| `string[]` | | A list of authorized parties. For example, `['http://localhost:4003', 'https://clerk.com']` |
+| `clockSkewInMs?` | `number` | Specifies the allowed time difference (in milliseconds) between the Clerk server (which generates the token) and the clock of the user's application server when validating a token. Defaults to 5000 ms (5 seconds).|
+| `jwtKey?` | `string` | The PEM public key from the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page -> Show JWT public key -> JWKS URL** section of the Clerk Dashboard. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables) instead. |
```
Comment (alexisintech) [inline docs/references/backend/verify-token.mdx:28]:
```suggestion
| `jwtKey?` | `string` | The JWT Public Key from the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page -> Show JWT public key -> JWKS URL** section of the Clerk Dashboard. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables) instead. |
```

File: docs/references/backend/verify-token.mdx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+> [!WARNING]
+> You must provide either `jwtKey` or `secretKey`.
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `audience?` | `string \| string[]` | A string or list of [audiences](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3). |
+| `authorizedParties?`| `string[]` | | A list of authorized parties. For example, `['http://localhost:4003', 'https://clerk.com']` |
+| `clockSkewInMs?` | `number` | Specifies the allowed time difference (in milliseconds) between the Clerk server (which generates the token) and the clock of the user's application server when validating a token. Defaults to 5000 ms (5 seconds).|
+| `jwtKey?` | `string` | The JWT Public Key from the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page -> Show JWT public key -> JWKS URL** section of the Clerk Dashboard. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables) instead. |
```
Comment (BRKalow) [inline docs/references/backend/verify-token.mdx:33]:
```suggestion
| `jwtKey?` | `string` | The JWT Public Key from the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page -> Show JWT public key -> PEM Public Key** section of the Clerk Dashboard. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables) instead. |
```

File: docs/references/backend/verify-token.mdx [hunk 3]
```diff
@@ -0,0 +1,108 @@
+---
+title: verifyToken()
+description: Use Clerk's Backend SDK to verify a token signature.
+---
+
+{/* JS page: https://github.com/clerk/javascript/blob/843e96e505cd91d676f0736e24465726b55014a7/packages/backend/src/tokens/verify.ts#L13 */}
+
+# `verifyToken()`
+
+Verifies a token signature.
```
Comment (BRKalow) [inline docs/references/backend/verify-token.mdx:10]:
```suggestion
Verifies a Clerk-generated token signature.
```

File: docs/references/backend/verify-token.mdx [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `audience?` | `string \| string[]` | A string or list of [audiences](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3). |
+| `authorizedParties?`| `string[]` | | A list of authorized parties. For example, `['http://localhost:4003', 'https://clerk.com']` |
+| `clockSkewInMs?` | `number` | Specifies the allowed time difference (in milliseconds) between the Clerk server (which generates the token) and the clock of the user's application server when validating a token. Defaults to 5000 ms (5 seconds).|
+| `jwtKey?` | `string` | The JWT Public Key from the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page -> Show JWT public key -> JWKS URL** section of the Clerk Dashboard. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables) instead. |
+| `secretKey?` | `string` | The Clerk secret key from the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page in the Clerk Dashboard.  |
+| `skipJwksCache?` | `boolean` | A flag to skip ignore cache and always fetch JWKs before each jwt verification. |
```
Comment (BRKalow) [inline docs/references/backend/verify-token.mdx:35]:
```suggestion
| `skipJwksCache?` | `boolean` | A flag to ignore the JWKS cache and always fetch JWKS before each jwt verification. |
```

File: docs/references/backend/sessions/authenticate-request.mdx [hunk 5]
```diff
@@ -33,11 +33,11 @@ function authenticateRequest: (request: Request, options: AuthenticateRequestOpt
...
+| `jwtKey?` | `string` | The PEM public key from the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page -> Show JWT public key -> JWKS URL** section of the Clerk Dashboard. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables) instead. |
 | `audience?` | `string \| string[]` | A string or list of [audiences](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3). |
 | `authorizedParties`| `string[]` | | A list of authorized parties. For example, `['http://localhost:4003', 'https://clerk.com']` |
 | `clockSkewInMs?` | `number` | Specifies the allowed time difference (in milliseconds) between the Clerk server (which generates the token) and the clock of the user's application server when validating a token. Defaults to 5000 ms (5 seconds).|
-| `jwksCacheTtlInMs?` | `number` | Specifies the allowed time (in milliseconds) the JWKs are considered valid in cache . Defaults to 3600_000 ms (1 hour).|
+| `jwksCacheTtlInMs?` (deprecated) | `number` | Deprecated. Specifying a cache TTL is now a no-op. |
 | `skipJwksCache?` | `boolean` | A flag to skip ignore cache and always fetch JWKs before each jwt verification.|
```
Comment (BRKalow) [inline docs/references/backend/sessions/authenticate-request.mdx:41]:
```suggestion
| `skipJwksCache?` | `boolean` | A flag to skip ignore cache and always fetch JWKS before each jwt verification.|
```

File: docs/references/backend/sessions/authenticate-request.mdx [hunk 6]
```diff
@@ -33,11 +33,11 @@ function authenticateRequest: (request: Request, options: AuthenticateRequestOpt
...
 | `signUpFallbackRedirectUrl?`         | `string`                                                                              | The fallback URL to redirect to after the user signs up, if there's no `redirect_url` in the path already. Defaults to `/`. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects) instead.                                                                                                                                                                            |
-| `jwtKey?` | `string` | The PEM public key from the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page -> Advanced -> JWT public key** section of the Clerk Dashboard. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables) instead. |
+| `jwtKey?` | `string` | The PEM public key from the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page -> Show JWT public key -> JWKS URL** section of the Clerk Dashboard. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables) instead. |
```
Comment (BRKalow) [inline docs/references/backend/sessions/authenticate-request.mdx:36]:
```suggestion
| `jwtKey?` | `string` | The PEM public key from the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page -> Show JWT public key -> PEM Public Key** section of the Clerk Dashboard. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables) instead. |
```

File: docs/references/backend/verify-token.mdx [hunk 7]
```diff
@@ -0,0 +1,108 @@
+---
+title: verifyToken()
+description: Use Clerk's Backend SDK to verify a token signature.
+---
+
+{/* JS page: https://github.com/clerk/javascript/blob/843e96e505cd91d676f0736e24465726b55014a7/packages/backend/src/tokens/verify.ts#L13 */}
+
+# `verifyToken()`
```
Comment (BRKalow) [inline docs/references/backend/verify-token.mdx:8]: I feel like we might want to call out that this is a lower-level method and intended for more advanced use-cases, in most cases we want to recommend that people use `authenticateRequest()` instead if they are securing a more traditional API.

**Review by jescalan — APPROVED** [review]
LGTM, agreed with bryce's comment though, if you have a `req` object, you could save a lot of code using `authenticateRequest`

**Comment by alexisintech** [comment]
> LGTM, agreed with bryce's comment though, if you have a `req` object, you could save a lot of code using `authenticateRequest`

i know! i added a callout referencing it, warning users that it's a lower level method. but we should still document it !

**Pull request merged**